### ISSUE TITLE
Failed to ensure reproduceable training

### ISSUE THREAD

**Issue by GYDDHPY** [282857341/nnFormer#100]
The deterministic settings used in nnformer seems failed to ensure reproduceable training.
Even I set the seed, the training loss of epochs can be different if I run the training again.
According to nnformer, the settings are as follow:
```
seed=1234567
np.random.seed(seed)
os.environ['PYTHONHASHSEED'] = str(seed)
np.random.seed(seed)
torch.manual_seed(seed)
torch.cuda.manual_seed(seed)
torch.cuda.manual_seed_all(seed)
torch.backends.cudnn.deterministic = True
torch.backends.cudnn.benchmark = False
```
`torch.backends.cudnn.enabled = False` is not suitable to my service, as it will cause error.

Could you please tell me how to guarantee reproduceable training?
Looking forward to your reply!
